TestDrive1/Program.cs

```diff
@@ -9,12 +11,20 @@ static void Main(string[] args)
             // Please keep your IE configuration settings:
             // 1. Check on "Enable Protected Mode" at ALL zones in "Security" tab of Internet Options dialog.
             // 2. Browser zoom level keep to 100%.
+            var ieOptions = new InternetExplorerOptions
+            {
+                // Uncomment these lines if you use Microsoft Edge IE mode.
+                // AttachToEdgeChrome = true,
+                // EdgeExecutablePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe",
+                IgnoreZoomLevel = true,
+            };
```

Comment:
`IgnoreZoomLevel = true` should be removed here? If it's OK to set `true`, above comments are unnecessary.